keps/1282-pods-ready-requeue-strategy/README.md

```diff
@@ -75,25 +105,69 @@ Consider including folks who also work outside the SIG or subproject.
 
 ### API Changes
 
-Add an additional field to the Kueue ConfigMap to allow administrators to specify what timestamp to consider during queue sorting (under the pre-existing waitForPodsReady block).
+#### KueueConfig
+
+Add an additional field to the KueueConfig to allow administrators to specify what timestamp to consider during queue sorting (under the pre-existing waitForPodsReady block).
 
 Possible settings:
 
 * `Eviction` (Back of queue)
 * `Creation` (Front of queue)
 
-```yaml
-kind: ConfigMap
-metadata:
-  name: kueue-manager-config
-  namespace: kueue-system
-data:
-  controller_manager_config.yaml: |
-    apiVersion: config.kueue.x-k8s.io/v1beta1
-    kind: Configuration
-    # ...
-    waitForPodsReady:
-      requeuingTimestamp: Creation | Eviction # <-- New field
+```go
+type WaitForPodsReady struct {
+	...
+	// requeuingStrategy defines the strategy for requeuing a Workload
+	// +optional
+	RequeuingStrategy *RequeuingStrategy `json:"requeuingStrategy,omitempty"`
+}
+
+type RequeuingStrategy struct {
+	// timestamp defines the timestamp used for requeuing a Workload
+	// that was evicted due to Pod readiness. Defaults to Eviction.
+	// +optional
+	Timestamp *RequeuingTimestamp `json:"timestamp,omitempty"`
+	
+	// backOffLimitCount defines the maximum number of requeuing retries.
+	// When the number is reached, the workload is deactivated.	
+	//
+	// Defaults to null. 
+	// +optional
+	BackOffLimitCount *int32 `json:"backOffLimitCount,omitempty"`
+    
+	// backOffLimitTimeout defines the time for a workload that 
+	// has once been admitted to reach the PodsReady=true condition. 
+	// When the time is reached, the workload is deactivated.
+	// 	
+	// Defaults to null.
+	// +optional
+	BackOffLimitTimeout *int32 `json:"backOffLimitTimeout,omitempty"`
```

Comment:
I don't understand what this is for. Can you rephrase?

Reply:
I will remove this knob as we discussed here: https://github.com/kubernetes-sigs/kueue/pull/1608#discussion_r1475161093

Reply:
Please remove so we can move forward with the KEP.

Reply:
Done.